include/circt/Dialect/FIRRTL/FIRRTLOps.h

```diff
@@ -51,7 +51,7 @@ size_t getNumPorts(Operation *op);
 bool isConstant(Operation *op);
 bool isConstant(Value value);
 
-/// Returns true if this is a 'const' type that can only hold compile-time
+/// Returns true if the type is a 'const' type that can only hold compile-time
 /// constant values
```

Comment:
Is "compile-time" the right wording? From the spec: `A constant type is a type whose value is guaranteed to be unchanging at circuit execution time`.

Reply:
Addressed in #5151